dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -37,7 +63,29 @@ void NavigationViewItem::OnApplyTemplate()
     // Retrieve pointers to stable controls 
     winrt::IControlProtected controlProtected = *this;
     m_helper.Init(controlProtected);
-    m_navigationViewItemPresenter.set(GetTemplateChildT<winrt::NavigationViewItemPresenter>(c_navigationViewItemPresenterName, controlProtected));
+
+    if (auto rootGrid = GetTemplateChildT<winrt::Grid>(c_rootGrid, controlProtected))
+    {
+        m_rootGrid.set(rootGrid);
+
```

Comment:
this means m_rootGird wont get set to null if GetTemplateChildT fails. I think the lambda version is slightly better for this reason, look at numberbox.cpp for example.  This would be okay to not fix though.

Reply:
m_rootGrid will be null when the object is created no ?

Reply:
Yep, that is the case. The lambda approach does seem better, however this is currently the approach we've been taking throughout this whole control. I think possibly creating a task to do a mass update might be a good idea.